### ISSUE TITLE
License

### ISSUE THREAD

**Issue by fabaff** [JasonCarter80/concord232#19]
Please add a license statement to the repo.

To create a RPM package for the Fedora Package Collection I need to know that license the code is under. Thanks.

**Comment by AZDane** [comment]
Yes, please add license statement. Its being pulled from Home Assistant because of this

**Comment by dotlambda** [comment]
duplicate of https://github.com/JasonCarter80/concord232/issues/17
